device-types/Ubiquiti/UACC-AI-Port-RM.yaml

```diff
@@ -0,0 +1,55 @@
+---
+manufacturer: Ubiquiti
+model: AI Port Rack Mount
+slug: ubiquiti-uacc-ai-port-rm
+part_number: UACC-AI-Port-RM
+u_height: 1
+is_full_depth: false
+airflow: passive
+subdevice_role: parent
+weight: 650
+weight_unit: g
+module-bays:
+  - name: '1'
+    label: AI Port Slot 1
+    position: '1'
+    description: Slot for UP-AI-Port module
+  - name: '2'
+    label: AI Port Slot 2
+    position: '2'
+    description: Slot for UP-AI-Port module
+  - name: '3'
+    label: AI Port Slot 3
+    position: '3'
+    description: Slot for UP-AI-Port module
+  - name: '4'
+    label: AI Port Slot 4
+    position: '4'
+    description: Slot for UP-AI-Port module
+  - name: '5'
+    label: AI Port Slot 5
+    position: '5'
+    description: Slot for UP-AI-Port module
+  - name: '6'
+    label: AI Port Slot 6
+    position: '6'
+    description: Slot for UP-AI-Port module
+front_image: true
+rear_image: true
+comments: |
```

Comment:
Thank you for your contribution,
Please reduce the amount of information in the comments property, this should only be a brief description with a link to the documentation.